integr8ly/integreatly-operator
# MGDAPI-5703 removing alerts installation, upgrade and missing metrics from openshift-monitoring namespace

# Issue link

MGDAPI-5703 

# What

removing alerts installation, upgrade and missing metrics from openshift-monitoring namespace
# Verification steps



- install the rhoam addon via the ocm ui
- Check the rhoam alerts are in the openshift monitoring namespace in the openshift-monitoring prometheus ui. You can use port-forward to access through http://localhost:9090/alerts?search=rhoam
```bash
oc port-forward -ns openshift-monitoring service/prometheus-operated 9090:9090
```
![image](https://github.com/integr8ly/integreatly-operator/assets/16667688/3402f0f3-0059-4f94-abfc-3e8e4d863cae)

- Patch the rhoam csv with this operator image `quay.io/austincunningham/managed-api-service: v1.39.0` 
- You should see the rhoam alerts are removed from the Prometheus monitoring. Only rhoam related alert present will be ElevatingClusterAdminRHOAMSRE which is unaffected by this change
![image](https://github.com/integr8ly/integreatly-operator/assets/16667688/071f54ee-9615-41e2-afad-b75bfc3aa601)
